cl/corpus_importer/tasks.py

```diff
@@ -1215,7 +1225,167 @@ def make_docket_by_iquery(
         d,
         tag_names,
         add_to_solr=True,
+        from_iquery_scrape=from_iquery_scrape,
+    )
+
+
+@retry((requests.Timeout, PacerLoginException), tries=3, delay=0.25, backoff=1)
+def query_iquery_page(
```

Comment:
Sure, in this case, I created a method called `query_case_query_report`, which encapsulates the common code so it can be reused by `query_iquery_page` and `make_docket_by_iquery_base`.